### DISCUSSION TITLE
Close Button of Left Menu

### DISCUSSION THREAD

**Discussion by aisagacn** [web-infra-dev/rspress#1214] [Ideas]
建议左侧菜单 添加关闭按钮；
方便 内容 宽屏查看；
类似 nextra

**Comment by Timeless0911** [comment]
Rspress 和 Vitepress 以及 Vuepress 一样，都采用响应式设计，在浏览器窗口宽度不够时，默认折叠菜单栏。

你也可以通过定制主题的方式，实现你的需求。
